experiments/opencog/pattern_matcher_vqa/pattern_matcher_vqa.py

```diff
@@ -344,11 +353,11 @@ def answerQuestionByImage(self, image, question, use_pm=True) -> QueryProcessing
                 queryInScheme = self.questionConverter.convertToOpencogSchemeURE(relexFormula)
             if queryInScheme is None:
                 self.logger.error('Question was not parsed')
-                return
+                return FailedProcessingData('Question was not parsed')
```

Comment:
May  be "Unsupported type of the question" ? "Question was not parsed" may mean some internal error, but here only reason is probably unsupported question type. I am not sure, what do you think?

Reply:
yes, is formula is correct then queryInScheme would be not None

Reply:
Then "Unsupported type of question" would be more clear for the end user I believe.